atm.js

```diff
@@ -1,21 +1,75 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$("input").eq(0).attr("id", "cInBox"); //create id for checking account input box
+$("input").eq(1).attr("id", "cDeBut"); //create id for checking deposit button
+$("input").eq(2).attr("id", "cWiBut"); //create id for checking withdraw button
+$("input").eq(3).attr("id", "sInBox"); //create id for savings account input box
+$("input").eq(4).attr("id", "sDeBut"); //create id for checking deposit button
+$("input").eq(5).attr("id", "sWiBut"); //create id for checking withdraw button
+$(".balance").eq(0).attr("id", "cBalTot");//create id for checking balance total
+$(".balance").eq(1).attr("id", "sBalTot");//create id for savings balance total
```

Comment:
I like that you're using IDs here!

That said, I'd suggest just adding these IDs to the HTML itself (unless we say otherwise, you're allowed to modify the HTML directly in an assignment).

Also, I'd suggest more descriptive ID names, such as `checking-deposit-btn`, `savings-input`, etc